web/src/ehr/patient/dossier/NewReport.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <modal-window
+      title="Create new Dossier report"
+      type="add"
+      :confirm-fn="submit"
+      confirm-text="Create Report"
+      :cancel-route="{name: 'ehr.patient', params: {id: this.$route.params.id}}">
+    <div class="mt-4">
+      <h1>New report</h1>
+      <form>
+        <div class="sticky top-0 z-10 p-3 bg-red-100 text-red-500 rounded-md" v-if="formErrors.length">
+          <label class="text-red-500">Please correct the following error{{
+              formErrors.length === 0 ? '' : 's'
+            }}:</label>
+
+          <ul class="text-sm">
+            <li v-for="error in formErrors">— {{ error }}</li>
+          </ul>
+        </div>
```

Comment:
Can you make a component out of this? Something like: `FormErrors` as we use this piece of code for each form.